### DISCUSSION TITLE
dotfx serve http://localhost:8080 not working

### DISCUSSION THREAD

**Discussion by silentdevnull** [dotnet/docfx#8264] [General]
**Operating System**: (Windows)

**DocFX v3 beta docfx-win7-x64-3.0.0-beta1.986+6c5eeaa2f0**: 

**Template used**: (`conceptual`)

**Steps to Reproduce**:

1.  .\docfx new conceptual -o myDocfxTest
2.  .\docfx.exe build myDocfxTest
results:
``` pwsh
Loading docsets...
Loading config...
Restoring dependencies...
Building...
build done in 1.59s
```
3.   .\docfx.exe serve myDocfxTest
results:
``` pwsh
Loading docsets...
Loading config...
Restoring dependencies...
Building...
build done in 1.59s
PS C:\Users\User\Downloads\docfx-win7-x64-3.0.0-beta1.986+6c5eeaa2f0> .\docfx.exe serve myDocfxTest
Hosting environment: Production
Content root path: C:\Users\thoma\Downloads\docfx-win7-x64-3.0.0-beta1.986+6c5eeaa2f0\
Now listening on: http://0.0.0.0:8080
Application started. Press Ctrl+C to shut down.
```
4.  Go to http://localhost:8080/ 
5. 

**Expected Behavior**:
See the default template.

**Actual Behavior**:
This localhost page can’t be foundNo webpage was found for the web address: http://localhost:8080/
HTTP ERROR 404

**Comment by arc9693** [comment]
@silentdevnull Please try opening up the same page in incognito mode.

**Comment by silentdevnull** [comment]
> @silentdevnull Please try opening up the same page in incognito mode.

I have tried private mode in edge and it was the same result.

**Comment by computamike** [comment]
Can confirm I have the same issue - have been able to confirm that there is a process listening on port 8080 - so that's at least something.

Will see what I get it I telnet directly to that service.
![image](https://user-images.githubusercontent.com/464876/133770610-51f64675-1306-472a-ae0a-74f4a9ec2167.png)

**Comment by computamike** [comment]
wasn't able to connect via Telnet - was able to connect via wget - got this : 
![image](https://user-images.githubusercontent.com/464876/133770847-e8254ca5-04cf-40ab-9c54-1bd6121f84e8.png)

this was taken from WSL running on my laptop

**Comment by KalleOlaviNiemitalo** [comment]
If you try to open http://localhost:8080/ without running `docfx serve` at all, do you get status 404 then too? If you do, that means there's a different web server interfering.
